lib/explorer/series.ex

```diff
@@ -2577,6 +2601,7 @@ defmodule Explorer.Series do
 
     * floats: #{Shared.inspect_dtypes(@float_dtypes, backsticks: true)}
     * integers: #{Shared.inspect_dtypes(@integer_types, backsticks: true)}
+    * `:decimal`. The result will be a float.
```

Comment:
For consistent, it should be the following, no?

```suggestion
    * decimals: the result will be a float
```

We need to mirror it below.